custom-tests.yaml

```diff
@@ -766,6 +766,37 @@ api:
       var instance = new FetchEvent('fetch', { request: req });
   FileReader:
     __base: var instance = new FileReader();
+  FileSystem:
+    __base: >-
+      window.webkitRequestFileSystem(TEMPORARY, 8*1024 /* 8KB */, function(fs) {
+        callback(fs);
+      }, function(event) {
+        fail(event.message);
+      });
+  FileSystemEntry:
+    __base: <%api.FileSystemDirectoryEntry:callback%>
```

Comment:
Can we get a `FileSystemEntry` instance from `DataTransferItem.prototype.webkitGetAsEntry()` or `HTMLInputElement.prototype.webkitEntries`?